web/src/app/browser/src/keymanEngine.ts

```diff
@@ -175,6 +175,19 @@ export default class KeymanEngine extends KeymanEngineBase<BrowserConfiguration,
 
     await super.init(totalOptions);
 
+    // Used by keymanweb.com; if no keyboard-cookie exists, we need this to trigger
+    // default-stub selection on mobile devices so that a keyboard - and thus, the
+    // globe key - are accessible.
+    //
+    // The `super` call above initializes `keyboardRequisitioner`, as needed here.
+    this.keyboardRequisitioner.cloudQueryEngine.once('unboundregister', () => {
```

Comment:
This event was added in #9304 to facilitate handling of keymanweb.com's direct call of `keyman.register`.